app/views/curation_concern/permissions/confirm.html.erb

```diff
@@ -3,19 +3,11 @@
     <h4>Apply changes to contents?<h4>
   </div>
   <div class="panel-body">
-    <% if curation_concern.under_embargo? %>
-      <p>You've applied an embargo to this <%= curation_concern.human_readable_type %>, <i><%= curation_concern %></i>, changing its visibility to <b><%= visibility_badge(curation_concern.visibility) %></b> until <%= curation_concern.embargo_release_date.to_date.to_formatted_s(:long_ordinal) %>.</p>
-      <p>Would you like to apply the same embargo to all of the files within the  <%= curation_concern.human_readable_type %> as well?</p>
-    <% elsif curation_concern.active_lease? %>
-      <p>You've applied a lease to this <%= curation_concern.human_readable_type %>, <i><%= curation_concern %></i>, changing its visibility to <b><%= visibility_badge(curation_concern.visibility) %></b> until <%= curation_concern.lease_expiration_date.to_date.to_formatted_s(:long_ordinal) %>.</p>
-      <p>Would you like to apply the same lease to all of the files within the  <%= curation_concern.human_readable_type %> as well?</p>
-    <% else %>
       <p>You've changed the permissions on this <%= curation_concern.human_readable_type %>, <i><%= curation_concern %></i>, making it visible to <b><%= visibility_badge(curation_concern.visibility) %></b>.</p>
       <p>Would you like change all of the files within the <%= curation_concern.human_readable_type %> to <b><%= visibility_badge(curation_concern.visibility) %></b> as well?</p>
-    <% end %>
   </div>
   <div class="form-actions panel-footer">
     <%= button_to "Yes please.", copy_curation_concern_permission_path(curation_concern), class: 'btn btn-primary' %>
-    <%= link_to "No. I'll update it manually.", polymorphic_path([:curation_concern, curation_concern]), class: 'btn' %>
+    <%= link_to "No. I'll update it manually.", Sufia::Engine.routes.url_helpers.generic_work_path(curation_concern), class: 'btn' %>
```

Comment:
@scherztc similar wonder to above...   I would not spend much time on it, but if [sufia, curation_concern] works it would be nice to be object agnostic.